main.js

```diff
@@ -20,15 +20,15 @@ const check_number = (value) => {
 const judge_fizzbuzz = () => {
     const fizznum = document.getElementById("fizznum").value;
     const buzznum = document.getElementById("buzznum").value;
-    resultText.innerHTML = "";
+    result.innerHTML = "";
     
     for(let counter = 1; counter < 100; counter++){
         if(counter % fizznum == 0 && counter % buzznum == 0){
```

Comment:
`{}`の両外側にはスペースをあけましょう。
その他の部分についても確認し修正しましょう。
```suggestion
        if(counter % fizznum == 0 && counter % buzznum == 0) {
```